qemu/tests/blockdev_inc_backup_inconsistent_bitmap.py

```diff
@@ -35,13 +35,15 @@ def is_image_bitmap_existed(self):
         out = self._data_image_obj.info()
         return out and self._bitmaps[0] in out
 
-    def check_bitmap_status(self, status):
+    def check_bitmap_field(self, **args):
         bitmap = get_bitmap_by_name(self.main_vm, self._source_nodes[0],
                                     self._bitmaps[0])
         if bitmap is None:
             self.test.fail('Failed to get bitmap %s' % self._bitmaps[0])
-        elif status != bitmap.get('status'):
-            self.test.fail('bitmap status changed')
+        else:
+            for key, value in args.items():
+                if value != bitmap[key]:
+                    self.test.fail('bitmap status changed')
```

Comment:
How about `'bitmap field '%s' is not correct: expected %s, got %s' %(key, value, bitmap[key])` ?

Reply:
@yanan-fu Updated. Thanks.